.github/workflows/test-ghcup.yaml

```diff
@@ -93,3 +84,39 @@ jobs:
 
       - if: runner.os == 'Windows'
         run: ghcup run -m sh -- -c 'pacman --version'
+
+  config:
+    strategy:
+      matrix:
+        runs-on:
+          - ubuntu-latest
+        version:
+          - latest
+    runs-on: ${{ matrix.runs-on }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./
+        with:
+          stack-hook: true
+          version: ${{ matrix.version }}
+          config: |
+            # see https://github.com/haskell/ghcup-hs/blob/master/data/config.yaml
+            # for full documentation
+            url-source:
+              - StackSetupURL
+            verbose: true
```

Comment:
Here... I'd like to be able to have this as an unknown object that we stringify. But `core.getInput` will fail.

Reply:
I think it's not possible reading https://github.com/actions/toolkit/issues/184